lib/protos.js

```diff
@@ -248,6 +249,7 @@ exports.load = function(name, locals, callback) {
   if (!template) throw new Error(fmt('template "%s" not defined.', name));
   var attrs = render(template, locals);
   callback = callback || noop;
+  shouldMoveIframeToBody = shouldMoveIframeToBody || false;
```

Comment:
:nit: you can do function(name, locals, callback, shouldMoveIframeToBody = false)

Same for the other PR.